lib/event/error_event.js

```diff
@@ -0,0 +1,39 @@
+/*! @license
+ * Shaka Player
+ * Copyright 2016 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+goog.provide('shaka.Player.ErrorEvent');
+
+goog.require('shaka.util.Error');
+goog.require('shaka.util.FakeEvent');
+
+
+/**
+ * @event shaka.Player.ErrorEvent
+ * @description Fired when a playback error occurs.
+ * @property {string} type
+ *   'error'
+ * @property {!shaka.util.Error} detail
+ *   An object which contains details on the error.  The error's
+ *   <code>category</code> and <code>code</code> properties will identify the
+ *   specific error that occurred.  In an uncompiled build, you can also use the
+ *   <code>message</code> and <code>stack</code> properties to debug.
+ * @extends {shaka.util.FakeEvent}
+ * @exportDoc
```

Comment:
Not sure, but should this be `@export` so it can be called by apps?

Reply:
If I change to @export the documentation is not generated correctly.... I don't know why...